5.2/5.2.2/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 5.2.2
+Создайте функцию readNumber, которая будет запрашивать ввод числового значения до тех пор, пока посетитель его не введёт.
+Функция должна возвращать числовое значение.
+Также надо разрешить пользователю остановить процесс ввода, отправив пустую строку или нажав «Отмена». В этом случае
+функция должна вернуть null.
+<script>
+	function readNumber() {
```

Comment:
this function works not very well because if I input spaces '    ', it will return these spaces '    ' as well. But in this case it should prompt for a message